sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -40,6 +40,14 @@ class DirectRunner(PipelineRunner):
   def __init__(self):
     self._cache = None
 
+  def apply_CombinePerKey(self, transform, pcoll):
+    from apache_beam.runners.direct.helper_transforms import LiftedCombinePerKey
```

Comment:
No, for the same reason the imports below can't be moved to the top. Copied the TODO.